certified-connectors/Adobe Creative Cloud Libraries/README.md

```diff
@@ -0,0 +1,25 @@
+# Adobe Creative Cloud Libraries
+Adobe Creative Cloud Libraries lets you connect Creative Cloud Libraries with thousands of the most popular apps so you can automate your work and have more time for what matters most-no code required.
+
```

Comment:
@dches Publisher section is missing, please add it.

Reply:
@sriyen-msft As I see in the description [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) - Publisher is required only for Independent-Publisher-Connector. Should I add this section for "certified-connector"?

Reply:
added publisher to the readme.md file